dockerfiles/centos/centos-7/centos-7-base/Dockerfile

```diff
@@ -51,12 +51,13 @@ RUN yum -y install epel-release && \
         tmux \
         sudo \
         which && \
+    cp -af /etc/skel/ /etc/vncskel/ && \
```

Comment:
no fluxbox on centos. means centos has a desktop but no wm so it will look worse.